packages/pieces/community/mollie/src/lib/actions/search-payment.ts

```diff
@@ -0,0 +1,50 @@
+import { createAction, Property } from '@activepieces/pieces-framework';
+import { HttpMethod } from '@activepieces/pieces-common';
+import { MollieAuth } from '../common/auth';
+import { makeRequest } from '../common/client';
+
+export const searchPayment = createAction({
+  auth: MollieAuth,
+  name: 'searchPayment',
+  displayName: 'Search Payment',
+  description: 'Search for payments based on filter criteria',
+  props: {
+    from: Property.ShortText({
+      displayName: 'From Payment ID',
+      description: 'Payment ID to start searching from (for pagination)',
+      required: false,
+    }),
+    limit: Property.Number({
+      displayName: 'Limit',
+      description: 'Number of payments to return (max 250, default 10)',
+      required: false,
+    }),
```

Comment:
Add support for sort per API docs.


```suggestion
    sort: Property.StaticDropdown({
  displayName: 'Sort',
  description: 'Order of results',
  required: false,
  options: { options: [{ label: 'Ascending', value: 'asc' }, { label: 'Descending', value: 'desc' }] },
}),
```